### PR TITLE
Chore: Gateway Vale cleanup

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Ran Vale on `app/gateway` and `app/_src/gateway` and fixed everything that came up.

Also ran into a few spots where I missed standardizing the hostname, so fixed those as well.

Vale should pass on this PR.

### Testing instructions

Preview link: Too many.

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by lena-larionova** [comment]
Note that I commented out the Developer Portal > Dev Portal rule and added rules for our existing variables instead. We can decide in the future if we should standardize on Dev Portal, but currently that rule only really applies to Konnect.

File: app/gateway/2.6.x/configure/auth/oidc-azuread.md [hunk 1]
```diff
@@ -129,5 +129,5 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 [add-certificate]: /gateway/{{page.kong_version}}/admin-api/#add-certificate
 [add-service]: /gateway/{{page.kong_version}}/admin-api/#service-object
 [oidc-id-token]: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.6.x/configure/auth/oidc-azuread.md:132]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

Comment (fabianrbz) [reply app/gateway/2.6.x/configure/auth/oidc-azuread.md:132]: totally unrelated to your changes, but It looks like this list is being rendered.
should it be a bullet list of links?

File: app/gateway/2.6.x/configure/auth/oidc-okta.md [hunk 2]
```diff
@@ -121,5 +121,5 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 [okta-register-app]: https://developer.okta.com/docs/guides/add-an-external-idp/openidconnect/register-app-in-okta/
 [add-certificate]: /gateway/latest/admin-api/#add-certificate
 [add-service]: /gateway/{{page.kong_version}}/admin-api/#service-object
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.6.x/configure/auth/oidc-okta.md:124]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

File: app/gateway/2.6.x/configure/graphql-quickstart.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ badge: enterprise
...
+{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```
Comment (fabianrbz) [inline app/gateway/2.6.x/configure/graphql-quickstart.md:8]:
```suggestion
{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](/hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```

File: app/gateway/2.6.x/pdk/kong.ip.md [hunk 4]
```diff
@@ -22,7 +24,7 @@ Trusted IPs module
  Trusted IP addresses are those that are known to send correct replacement
  addresses for clients (as per the chosen header field, e.g. X-Forwarded-*).
 
- See [docs.konghq.com/latest/configuration/#trusted_ips](https://docs.konghq.com/latest/configuration/#trusted_ips)
+ See [docs.konghq.com/latest/configuration/#trusted_ips](latest/configuration/#trusted_ips)
```
Comment (fabianrbz) [inline app/gateway/2.6.x/pdk/kong.ip.md:27]:
```suggestion
 See [docs.konghq.com/latest/configuration/#trusted_ips](/latest/configuration/#trusted_ips)
```

File: app/gateway/2.6.x/plan-and-deploy/dns-considerations.md [hunk 5]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
 [mdn-cookies]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies
 [mdn-set-cookie]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
 [mdn-cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-[oidc-plugin]: https://docs.konghq.com/hub/kong-inc/openid-connect/
-[session-plugin]: https://docs.konghq.com/hub/kong-inc/session/
+[oidc-plugin]: hub/kong-inc/openid-connect/
```
Comment (fabianrbz) [inline app/gateway/2.6.x/plan-and-deploy/dns-considerations.md:256]:
```suggestion
[oidc-plugin]: /hub/kong-inc/openid-connect/
```

File: app/gateway/2.6.x/plan-and-deploy/dns-considerations.md [hunk 6]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
...
+[session-plugin]: hub/kong-inc/session/
```
Comment (fabianrbz) [inline app/gateway/2.6.x/plan-and-deploy/dns-considerations.md:257]:
```suggestion
[session-plugin]: /hub/kong-inc/session/
```

File: app/gateway/2.6.x/reference/external-plugins.md [hunk 7]
```diff
@@ -627,7 +627,7 @@ CMD ["kong", "docker-start"]
 [go-pluginserver-makefile]: https://github.com/Kong/go-pluginserver/blob/master/Makefile
 [go-plugins]: https://github.com/Kong/go-plugins
 [go-pdk]: https://github.com/Kong/go-pdk
-[kong-pdk]: https://docs.konghq.com/latest/plugin-development/
+[kong-pdk]: latest/plugin-development/
```
Comment (fabianrbz) [inline app/gateway/2.6.x/reference/external-plugins.md:630]:
```suggestion
[kong-pdk]: /latest/plugin-development/
```

File: app/gateway/2.6.x/vitals/index.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ Vitals is enabled by default in {{site.ee_product_name}} and available upon the
 
 You will need one of the following databases to use Vitals:
 * InfluxDB
-* PostgresSQL 9.5+
+* PostgreSQLSQL 9.5+
```
Comment (fabianrbz) [inline app/gateway/2.6.x/vitals/index.md:19]:
```suggestion
* PostgreSQL 9.5+
```

File: app/gateway/2.7.x/configure/auth/oidc-azuread.md [hunk 9]
```diff
@@ -129,5 +129,5 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 [add-certificate]: /gateway/{{page.kong_version}}/admin-api/#add-certificate
 [add-service]: /gateway/{{page.kong_version}}/admin-api/#service-object
 [oidc-id-token]: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.7.x/configure/auth/oidc-azuread.md:132]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

File: app/gateway/2.7.x/configure/auth/oidc-okta.md [hunk 10]
```diff
@@ -253,4 +253,4 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 
 [okta-authorization-server]: https://developer.okta.com/docs/guides/customize-authz-server/create-authz-server/
 [okta-register-app]: https://developer.okta.com/docs/guides/add-an-external-idp/openidconnect/register-app-in-okta/
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.7.x/configure/auth/oidc-okta.md:256]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

File: app/gateway/2.7.x/configure/graphql-quickstart.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ badge: enterprise
...
+{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```
Comment (fabianrbz) [inline app/gateway/2.7.x/configure/graphql-quickstart.md:8]:
```suggestion
{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](/hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```

File: app/gateway/2.7.x/install-and-run/upgrade-oss.md [hunk 12]
```diff
@@ -43,7 +43,7 @@ repository contains [openresty-build-tools](https://github.com/Kong/kong-build-t
 which allows you to more easily build OpenResty with the necessary patches and modules.
 
 There is a new way to deploy Go using Plugin Servers.
-For more information, see [Developing Go plugins](https://docs.konghq.com/gateway/2.6.x/reference/external-plugins/#developing-go-plugins).
+For more information, see [Developing Go plugins](gateway/2.6.x/reference/external-plugins/#developing-go-plugins).
```
Comment (fabianrbz) [inline app/gateway/2.7.x/install-and-run/upgrade-oss.md:46]:
```suggestion
For more information, see [Developing Go plugins](/gateway/2.6.x/reference/external-plugins/#developing-go-plugins).
```

File: app/gateway/2.7.x/install-and-run/upgrade-oss.md [hunk 13]
```diff
@@ -186,7 +186,7 @@ $ kong migrations up [-c configuration_file]
 
 If the command is successful, and no migration ran
 (no output), then you only have to
-[reload](https://docs.konghq.com/gateway/2.7.x/reference/cli/#kong-reload) Kong:
+[reload](gateway/2.7.x/reference/cli/#kong-reload) Kong:
```
Comment (fabianrbz) [inline app/gateway/2.7.x/install-and-run/upgrade-oss.md:189]:
```suggestion
[reload](/gateway/2.7.x/reference/cli/#kong-reload) Kong:
```

File: app/gateway/2.7.x/pdk/kong.ip.md [hunk 14]
```diff
@@ -22,7 +24,7 @@ Trusted IPs module
  Trusted IP addresses are those that are known to send correct replacement
  addresses for clients (as per the chosen header field, e.g. X-Forwarded-*).
 
- See [docs.konghq.com/latest/configuration/#trusted_ips](https://docs.konghq.com/latest/configuration/#trusted_ips)
+ See [docs.konghq.com/latest/configuration/#trusted_ips](latest/configuration/#trusted_ips)
```
Comment (fabianrbz) [inline app/gateway/2.7.x/pdk/kong.ip.md:27]:
```suggestion
 See [docs.konghq.com/latest/configuration/#trusted_ips](/latest/configuration/#trusted_ips)
```

File: app/gateway/2.7.x/plan-and-deploy/dns-considerations.md [hunk 15]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
 [mdn-cookies]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies
 [mdn-set-cookie]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
 [mdn-cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-[oidc-plugin]: https://docs.konghq.com/hub/kong-inc/openid-connect/
-[session-plugin]: https://docs.konghq.com/hub/kong-inc/session/
+[oidc-plugin]: hub/kong-inc/openid-connect/
```
Comment (fabianrbz) [inline app/gateway/2.7.x/plan-and-deploy/dns-considerations.md:256]:
```suggestion
[oidc-plugin]: /hub/kong-inc/openid-connect/
```

File: app/gateway/2.7.x/plan-and-deploy/dns-considerations.md [hunk 16]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
...
+[session-plugin]: hub/kong-inc/session/
```
Comment (fabianrbz) [inline app/gateway/2.7.x/plan-and-deploy/dns-considerations.md:257]:
```suggestion
[session-plugin]: /hub/kong-inc/session/
```

File: app/gateway/2.7.x/reference/external-plugins.md [hunk 17]
```diff
@@ -627,7 +627,7 @@ CMD ["kong", "docker-start"]
 [go-pluginserver-makefile]: https://github.com/Kong/go-pluginserver/blob/master/Makefile
 [go-plugins]: https://github.com/Kong/go-plugins
 [go-pdk]: https://github.com/Kong/go-pdk
-[kong-pdk]: https://docs.konghq.com/latest/plugin-development/
+[kong-pdk]: latest/plugin-development/
```
Comment (fabianrbz) [inline app/gateway/2.7.x/reference/external-plugins.md:630]:
```suggestion
[kong-pdk]: /latest/plugin-development/
```

File: app/gateway/2.7.x/vitals/index.md [hunk 18]
```diff
@@ -20,7 +20,7 @@ Vitals is enabled by default in {{site.ee_product_name}} and available upon the
 
 You will need one of the following databases to use Vitals:
 * InfluxDB
-* PostgresSQL 9.5+
+* PostgreSQLSQL 9.5+
```
Comment (fabianrbz) [inline app/gateway/2.7.x/vitals/index.md:23]:
```suggestion
* PostgreSQL 9.5+
```

File: app/gateway/2.8.x/configure/auth/oidc-azuread.md [hunk 19]
```diff
@@ -129,5 +129,5 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 [add-certificate]: /gateway/{{page.kong_version}}/admin-api/#add-certificate
 [add-service]: /gateway/{{page.kong_version}}/admin-api/#service-object
 [oidc-id-token]: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.8.x/configure/auth/oidc-azuread.md:132]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

File: app/gateway/2.8.x/configure/auth/oidc-okta.md [hunk 20]
```diff
@@ -253,4 +253,4 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 
 [okta-authorization-server]: https://developer.okta.com/docs/guides/customize-authz-server/create-authz-server/
 [okta-register-app]: https://developer.okta.com/docs/guides/add-an-external-idp/openidconnect/register-app-in-okta/
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```
Comment (fabianrbz) [inline app/gateway/2.8.x/configure/auth/oidc-okta.md:256]:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

File: app/gateway/2.8.x/configure/graphql-quickstart.md [hunk 21]
```diff
@@ -5,7 +5,7 @@ badge: enterprise
...
+{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```
Comment (fabianrbz) [inline app/gateway/2.8.x/configure/graphql-quickstart.md:8]:
```suggestion
{{site.base_gateway}} is an API gateway and platform. That means it is a form of middleware between computing clients and your API-based applications. {{site.base_gateway}} quickly and consistently extends the features of your APIs. Some of the popular features deployed through {{site.base_gateway}} include authentication, security, traffic control, serverless, analytics & monitoring, request/response transformations, and logging. To learn more about these features, see the [Hub page](/hub/) for plugins. For more about the benefits of Kong in general, please see the [FAQ](https://konghq.com/faqs).
```

File: app/gateway/2.8.x/install-and-run/helm-quickstart-enterprise.md [hunk 22]
```diff
@@ -308,7 +308,7 @@ Once all dependencies are installed and ready, deploy {{site.base_gateway}} to y
 
 For local deployments, Kong Manager is locally accessible at `https://kong.127-0-0-1.nip.io`. The [nip.io](https://nip.io) service resolves this domain to localhost also known as `127.0.0.1`.
 
-You can configure Kong via the Admin API with [decK](https://docs.konghq.com/deck/latest/), [Insomnia](https://docs.insomnia.rest/insomnia/get-started), HTTPie, or cURL, at `https://kong.127-0-0-1.nip.io/api`:
+You can configure Kong via the Admin API with [decK](deck/latest/), [Insomnia](https://docs.insomnia.rest/insomnia/get-started), HTTPie, or cURL, at `https://kong.127-0-0-1.nip.io/api`:
```
Comment (fabianrbz) [inline app/gateway/2.8.x/install-and-run/helm-quickstart-enterprise.md:311]:
```suggestion
You can configure Kong via the Admin API with [decK](/deck/latest/), [Insomnia](https://docs.insomnia.rest/insomnia/get-started), HTTPie, or cURL, at `https://kong.127-0-0-1.nip.io/api`:
```

File: app/gateway/2.8.x/pdk/kong.client.md [hunk 23]
```diff
@@ -58,9 +60,9 @@ Returns the remote address of the client making the request.  Unlike
  returns a forwarded address or not depends on several Kong configuration
  parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.client.md:64]:
```suggestion
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
```

File: app/gateway/2.8.x/pdk/kong.client.md [hunk 24]
```diff
@@ -58,9 +60,9 @@ Returns the remote address of the client making the request.  Unlike
...
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.client.md:65]:
```suggestion
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.client.md [hunk 25]
```diff
@@ -122,9 +124,9 @@ Returns the remote port of the client making the request.  Unlike
  when a load balancer is in front of Kong. Whether this function returns a
  forwarded port or not depends on several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.client.md:127]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
```

File: app/gateway/2.8.x/pdk/kong.client.md [hunk 26]
```diff
@@ -122,9 +124,9 @@ Returns the remote port of the client making the request.  Unlike
...
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.client.md:128]:
```suggestion
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
```

File: app/gateway/2.8.x/pdk/kong.client.md [hunk 27]
```diff
@@ -122,9 +124,9 @@ Returns the remote port of the client making the request.  Unlike
...
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.client.md:129]:
```suggestion
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.ip.md [hunk 28]
```diff
@@ -21,7 +23,7 @@ Trusted IPs module.
  addresses for clients (as per the chosen header field, for example
  X-Forwarded-*).
 
- See the [documentation on trusted IPs](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips).
+ See the [documentation on trusted IPs](gateway/latest/reference/configuration/#trusted_ips).
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.ip.md:26]:
```suggestion
 See the [documentation on trusted IPs](/gateway/latest/reference/configuration/#trusted_ips).
```

File: app/gateway/2.8.x/plan-and-deploy/dns-considerations.md [hunk 29]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
 [mdn-cookies]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies
 [mdn-set-cookie]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
 [mdn-cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-[oidc-plugin]: https://docs.konghq.com/hub/kong-inc/openid-connect/
-[session-plugin]: https://docs.konghq.com/hub/kong-inc/session/
+[oidc-plugin]: hub/kong-inc/openid-connect/
```
Comment (fabianrbz) [inline app/gateway/2.8.x/plan-and-deploy/dns-considerations.md:256]:
```suggestion
[oidc-plugin]: /hub/kong-inc/openid-connect/
```

File: app/gateway/2.8.x/plan-and-deploy/dns-considerations.md [hunk 30]
```diff
@@ -253,5 +253,5 @@ help Kong Support pinpoint the cause. Please provide it in tickets if possible.
...
+[session-plugin]: hub/kong-inc/session/
```
Comment (fabianrbz) [inline app/gateway/2.8.x/plan-and-deploy/dns-considerations.md:257]:
```suggestion
[session-plugin]: /hub/kong-inc/session/
```

File: app/gateway/2.8.x/vitals/index.md [hunk 31]
```diff
@@ -19,7 +19,7 @@ Vitals is enabled by default in {{site.ee_product_name}} and available upon the
 
 You can use one of the following storage strategies with Vitals:
 * Kong database:
-  * PostgresSQL 9.5+
+  * PostgreSQLSQL 9.5+
```
Comment (fabianrbz) [inline app/gateway/2.8.x/vitals/index.md:22]:
```suggestion
  * PostgreSQL 9.5+
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 32]
```diff
@@ -103,9 +105,9 @@ Returns the scheme component of the request's URL, but also considers
  Whether this function considers `X-Forwarded-Proto` or not depends on
  several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:110]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 33]
```diff
@@ -138,9 +140,9 @@ Returns the host component of the request's URL or the value of the "host"
  Whether this function considers `X-Forwarded-Host` or not depends on
  several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:145]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 34]
```diff
@@ -172,16 +174,16 @@ Returns the port component of the request's URL, but also considers
...
 
  **Note**: Kong does not offer support for the Forwarded HTTP Extension
  (RFC 7239) since it is not supported by ngx_http_realip_module.
 
  When running Kong behind the L4 port mapping (or forwarding), you can also
  configure:
- * [port\_maps](https://docs.konghq.com/gateway/latest/reference/configuration/#port_maps)
+ * [port\_maps](gateway/latest/reference/configuration/#port_maps)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:186]:
```suggestion
 * [port\_maps](/gateway/latest/reference/configuration/#port_maps)
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 35]
```diff
@@ -172,16 +174,16 @@ Returns the port component of the request's URL, but also considers
  Whether this function considers `X-Forwarded-Proto` or not depends on
  several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:179]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 36]
```diff
@@ -213,9 +215,9 @@ Returns the path component of the request's URL, but also considers
  Whether this function considers `X-Forwarded-Path` or not depends on
  several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:220]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

File: app/gateway/2.8.x/pdk/kong.request.md [hunk 37]
```diff
@@ -253,9 +255,9 @@ Returns the prefix path component of the request's URL that Kong stripped
  Whether this function considers `X-Forwarded-Prefix` or not depends on
  several Kong configuration parameters:
 
- * [trusted\_ips](https://docs.konghq.com/gateway/latest/reference/configuration/#trusted_ips)
- * [real\_ip\_header](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_header)
- * [real\_ip\_recursive](https://docs.konghq.com/gateway/latest/reference/configuration/#real_ip_recursive)
+ * [trusted\_ips](gateway/latest/reference/configuration/#trusted_ips)
+ * [real\_ip\_header](gateway/latest/reference/configuration/#real_ip_header)
+ * [real\_ip\_recursive](gateway/latest/reference/configuration/#real_ip_recursive)
```
Comment (fabianrbz) [inline app/gateway/2.8.x/pdk/kong.request.md:260]:
```suggestion
 * [trusted\_ips](/gateway/latest/reference/configuration/#trusted_ips)
 * [real\_ip\_header](/gateway/latest/reference/configuration/#real_ip_header)
 * [real\_ip\_recursive](/gateway/latest/reference/configuration/#real_ip_recursive)
```

**Review by fabianrbz — CHANGES_REQUESTED** [review]
❤️ ❤️ ❤️ ❤️ Thanks!
I left a few comments and suggestions, but otherwise, it looks great! 🎉

**Comment by lena-larionova** [comment]
Sorry for the mess of search & replace typos - two typos, but such far reach 😓

**Comment by lena-larionova** [comment]
Fixed all the real broken links.

**Review by fabianrbz — APPROVED** [review]
LGTM 🎉

**Pull request merged**